packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -11,6 +11,14 @@
     <oui-tile-definition
         data-term="{{ ::'vps_dashboard_tile_configuration_vcore' | translate }}"
     >
+        <oui-tile-description data-ng-if="$ctrl.statistics">
+            <progress
```

Comment:
No direct `oui-progress` possible here ?
(If `statistics` is meant to change in this page, it's ok)

Reply:
if i put directly `oui-progress` without `oui-tile-description`, it isn't interpreted end isn't in the DOM. :/